requirements-dev.txt

```
@@ -1,6 +1,6 @@
 autopep8
 click
-numpy==1.21.5
+numpy==1.22.0
```

```suggestion
numpy==1.22.0; python_version >= "3.8"
numpy==1.21.5; python_version < "3.8"
```
nit: The fastest way is removing Python 3.7 (EOL is 27 Jun 2023). Or, perhaps we can just make Python 3.7 still use numpy==1.21.5 for now.